source/core/OptOutCollection.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Xml.Serialization;
+
+namespace com.esendex.sdk.core
+{
+    /// <summary>
+    /// Represents a collecion of optout messages.
+    /// </summary>
+    [Serializable]
+    [XmlRoot("optouts", Namespace = Constants.API_NAMESPACE)]
```

Comment:
Because it allowed me to utilise the paging that has already been built for inbox. Json wouldn't work.